packages/wonder-blocks-breadcrumbs/src/components/breadcrumbs.js

```diff
@@ -36,6 +36,30 @@ const StyledList = addStyle("ol");
  *
  * 1. Breadcrumbs Item: Represents a section within the page.
  * 2. Separator: Adds a separator between each item.
+ *
+ * NOTE: `<BreadcrumbsItem />` only accepts two element types:
+ *
+ * 1. `string`
+ * 2. `<Link />`
+ *
+ * ```jsx
+ * import {
+ *     Breadcrumbs,
+ *     BreadcrumbsItem
+ * } from "@khanacademy/wonder-blocks-breadcrumbs";
+ *
+ * <Breadcrumbs>
+ *     <BreadcrumbsItem>
+ *         <Link href="">Course</Link>
+ *     </BreadcrumbsItem>
+ *     <BreadcrumbsItem>
+ *         <Link href="">Unit</Link>
+ *     </BreadcrumbsItem>
+ *     <BreadcrumbsItem>
+ *         <Link href="">Lesson</Link>
+ *     </BreadcrumbsItem>
+ * </Breadcrumbs>
+ * ```
```

Comment:
Ohhh I see that the bundle file size was increased ~145b due to these new comments. I still think this shouldn't count as a version bump as we are not introducing any changes to the code itself.

Reply:
@jeresig ☝️ This is the reason for the file size increase.

Reply:
But why do the comments increase the file size - don't we strip the comments out when generating the bundles? I thought the bundles were minified ahead of time. At the very least, I think we should do that if we're comparing the output to get a more accurate view of what changes are being made and how that might impact users. Out of scope of this PR, but would be good to look in to this in the future!

Reply:
It turns out we are not stripping the comments out :|. I've created this ticket to track that change: https://khanacademy.atlassian.net/browse/WB-1299

Reply:
Thank you!